app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+    def user_avatar(user, size=40)
```

Comment:
I love the use of gravatar if user has no avatar uploaded. That being said you should move this helper to `users_helper.rb` because it belongs to users and also indentation is broken it uses 4spaces instead of 2